docs/sources/installation/mac.md

```diff
@@ -1,116 +1,315 @@
 
-## Installation
+## Learn the key concepts before installing
+ 
+In a Docker installation on Linux, your machine is both the localhost and the
+Docker host. In networking, localhost means your computer. The Docker host is
+the machine on which the containers run. 
 
-1. Download the latest release of the [Docker for OS X Installer](
-   https://github.com/boot2docker/osx-installer/releases/latest) (Look for the
-   green Boot2Docker-x.x.x.pkg button near the bottom of the page.)
+On a typical Linux installation, the Docker client, the Docker daemon, and any
+containers run directly on your localhost. This means you can address ports on a
+Docker container using standard localhost addressing such as `localhost:8000` or
+`0.0.0.0:8376`.
 
-2. Run the installer by double-clicking the downloaded package, which will install a
-VirtualBox VM, Docker itself, and the Boot2Docker management tool.
-   ![](/installation/images/osx-installer.png)
+![Linux Architecture Diagram](/installation/images/linux_docker_host.png)
 
-3. Locate the `Boot2Docker` app in your `Applications` folder and run it.
-   Or, you can initialize Boot2Docker from the command line by running:
+In an OS X installation, the `docker` daemon is running inside a Linux virtual
+machine provided by Boot2Docker.
 
-	     $ boot2docker init
-	     $ boot2docker start
-	     $ $(boot2docker shellinit)
+![OSX Architecture Diagram](/installation/images/mac_docker_host.png)
 
-A terminal window will open and you'll see the virtual machine starting up. 
-Once you have an initialized virtual machine, you can control it with `boot2docker stop`
-and `boot2docker start`.
+In OS X, the Docker host address is the address of the Linux VM.
+When you start the `boot2docker` process, the VM is assigned an IP address. Under
+`boot2docker` ports on a container map to ports on the VM. To see this in
+practice, work through the exercises on this page.
 
-> **Note:**
-> If you see a message in the terminal that looks something like this:
->
->    `To connect the Docker client to the Docker daemon, please set: export 
-DOCKER_HOST=tcp://192.168.59.103:2375`
-> 
-you can safely set the environment variable as instructed.
 
-View the
-[Boot2Docker ReadMe](https://github.com/boot2docker/boot2docker/blob/master/README.md)
-for more information.
+## Install Boot2Docker
+ 
+1. Go to the [boo2docker/osx-installer ](
+https://github.com/boot2docker/osx-installer/releases/latest) release page.
 
-## Upgrading
+4. Download Boot2Docker by clicking `Boot2Docker-x.x.x.pkg` in the "Downloads"
+section.
 
-1. Download the latest release of the [Docker for OS X Installer](
-   https://github.com/boot2docker/osx-installer/releases/latest)
+3. Install Boot2Docker by double-clicking the package.
 
-2. If Boot2Docker is currently running, stop it with `boot2docker stop`. Then, run
-the installer package, which will update Docker and the Boot2Docker management tool.
+	The installer places Boot2Docker in your "Applications" folder.
```

Comment:
I htink the way you refer to it later in this doc is better

`the "Boot2Docker" application ...`